### ISSUE TITLE
The Example_OLR_Tracking_satellite notebook fails with latest `main`

### ISSUE THREAD

**Issue by simonrp84** [tobac-project/tobac#128]
Hi,
The `examples/`Example_OLR_Tracking_satellite.ipynb` file no longer produces the expected output.
This section of code:
```
# Plot map with all individual tracks:
import cartopy.crs as ccrs
fig_map,ax_map=plt.subplots(figsize=(10,10),subplot_kw={'projection': ccrs.PlateCarree()})
ax_map=tobac.map_tracks(Track,axis_extent=axis_extent,axes=ax_map)
```

Produces a map in which one of the trajectories has gone haywire:
![download](https://user-images.githubusercontent.com/13449576/167303949-95040aab-13e8-477a-bd94-44ae6b445a25.png)

I also see that the features returned by `tobac.feature_detection_multithreshold` and then subsequently by `tobac.segmentation_2D` are identical, not sure if that's intended.

I'm running a fully updated conda environment with python 3.9 and the latest tobac version on the `main` branch.

**Comment by freemansw1** [comment]
I believe that this is related to #121, where switching marking untracked cells as `-1` instead of `np.nan` resulted in the plotting function breaking. 

I'm going to suggest that we make a hotfix for this and release it as v1.3.1. I should be able to do a PR for this by the end of the week. In the meantime, I will add this to the 1.3.1 milestone.

**Comment by freemansw1** [comment]
I should have also added that the features being identical between `feature_detection_multithreshold` and `segmentation_2D` is expected behavior. Segmentation will only add additional information; ideally the `num` column. However, that is broken for most cases as discussed in #25

**Comment by freemansw1** [comment]
This is resolved in #130, and will be released as part of 1.3.1 fairly soon.
